packages/framework-types/src/concepts/register.ts

```diff
@@ -44,6 +45,15 @@ export class Register {
     return this
   }
 
+  /**
+   * Register a list of notifications to be added to the event-store on handler completion
+   * @param notifications
+   */
+  public notifications(...notifications: Array<NotificationInterface>): Register {
+    this.eventList.push(...notifications)
+    return this
```

Comment:
Why not use the same `register.events` method to register notifications? It seems a bit confusing to me that we have a separate way to handle them when at the end of the day, they're also events... what do you think?